packages/asset-uploader/src/helpers/array-buffer-helpers.ts

```diff
@@ -0,0 +1,27 @@
+import path from "path";
+
+export const mergeUint8Arrays = (a: Uint8Array, b: Uint8Array) => {
+  const c = new Uint8Array(a.length + b.length);
+  c.set(a);
+  c.set(b, a.length);
+
+  return c;
+};
+
+export const getFilenameAndExtension = (filename: string) => {
+  return [
```

Comment:
check out path.basename() and path.extname(), just use it where you need this, no need for custom utility, at least I don't see a point

Reply:
Path.basename returns the whole file name and i am already using path.extname